cpg-language-python/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonFrontendTest.kt

```diff
@@ -818,17 +816,22 @@ class PythonFrontendTest : BaseTest() {
         val p = tu.namespaces["issue615"]
         assertNotNull(p)
 
-        assertEquals(1, p.declarations.size)
+        assertEquals(
+            5,
+            p.variables.size
+        ) // including one dummy variable introduced for the loop var
+        assertEquals(
+            4,
+            p.variables.filter { !it.name.localName.contains("loopMultiVarHelperVar") }.size
```

Comment:
Any chance getting this constant from the StatementHandler? Maybe move it to PythonLanguage?